src/main/java/Task.java

```diff
@@ -0,0 +1,30 @@
+public class Task {
+    protected String description;
```

Comment:
Maybe a clearer name for description would be taskDescription. (applicable change to other classes as well)